src/style.css

```diff
@@ -1,3 +1,88 @@
+:root {
+  --background-color: #fff;
+  --main-color: #00acdc;
+  --border-height: 3px;
+  --font-size: 12px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  display: block;
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: var(--background-color);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.menu_box {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.menu_item {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  list-style: none;
+}
+
+.menu_link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  position: relative;
+}
+
+.menu_item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.menu_link:hover {
+  color: var(--main-color);
+}
+
+.is-active {
+  color: var(--main-color);
+}
+
+.is-active::after {
+  content: ' ';
+  position: absolute;
+  width: 100%;
+  bottom: 0;
+  height: 4px;
```

Comment:
The `height` of the `.is-active::after` pseudo-element is set to `4px`, but there is a custom property `--border-height` defined which is set to `3px`. To maintain consistency and reuse custom properties, consider using `var(--border-height)` here.